spec/models/zip_code_spec.rb

```diff
@@ -7,6 +7,30 @@
     it { should validate_presence_of(:d_codigo) }
   end
 
+  def create_states(states)
```

Comment:
I like the idea behind this methods, I would just suggest you to move them into the `support` directory this way you can reuse them on other tests, and we don't fill this spec file with so much noise :pray:

Reply:
Done!